frontend/src/infra/project/getProjects.ts

```diff
@@ -0,0 +1,12 @@
+import { apiClient } from '@/infra/lib/apiClient'
+import { Project } from '@/domain/Project'
+
+export const getProjects = async (): Promise<Array<Project>> => {
+  const response = await apiClient('/v1/projects/')
+
+  if (!response.ok) {
```

Comment:
Is the plan to bubble up error messages to the user as we evolve this code?

Reply:
Yes! thats the plan, this throw is just a placeholder for now